spec/graphql/persisted_queries/schema_patch_spec.rb

```diff
@@ -14,6 +14,24 @@ def call(error)
     end
   end
 
+  Tracer = Class.new do
```

Comment:
Currently. the only tests that I added for this are these integration-y tests.  It felt awkward testing it in each of the adapters, so I haven't added any there.  From the previous comment, if we move most of the implementation to the `BaseStorageAdapter`, it would make more sense to add a few tests there.  What do you think?

Reply:
We could probably move tests to the shared context and include for both adapter, but overall I agree that tracing should be tested separetely

Reply:
Since the implementation moved to `BaseStoreAdapter`, I've added the tests there and they seem like they are in the right place to me.